packages/runtime/container-runtime/src/deltaManagerSummarizerProxy.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDeltaManager, ReadOnlyInfo } from "@fluidframework/container-definitions";
+import { DeltaManagerProxyBase } from "@fluidframework/container-utils";
+import { IDocumentMessage, ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
+import { summarizerClientType } from "./summary";
+
+/**
+ * Proxy to the real IDeltaManager for restricting certain access to layers below container runtime in summarizer clients:
+ * - Summarizer client should be read-only to layers below the container runtime to restrict local changes.
+ * - Summarizer client should not be active to layers below the container runtime to restrict local changes.
+ */
+export class DeltaManagerSummarizerProxy
+	extends DeltaManagerProxyBase
+	implements IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>
+{
+	public get active(): boolean {
+		// Summarize clients should not be active. There shouldn't be any local changes (writes) in the summarizer
+		// except for the SummarizeOp which is generated by the runtime.
+		return this.isSummarizerClient ? false : this.deltaManager.active;
+	}
+
+	public get readOnlyInfo(): ReadOnlyInfo {
+		// Summarizer clients should be read-only as far as the runtime and layers below are concerned. There shouldn't
+		// be any local changes (writes) in the summarizer except for the SummarizeOp which is generated by the runtime.
+		if (this.isSummarizerClient) {
+			return {
+				readonly: true,
+				forced: false,
+				permissions: true,
```

Comment:
I'd leave this undefined... not sure who uses it or for what, but seems unnecessary to go so far as to say that the user has read-only permissions.

Reply:
Or maybe pass-through whatever the real DM says.

Reply:
I would say that the user does have read-only permission because it is a sytem client.

Reply:
Hmm I kind of see what you mean.  But the authenticated user does have permissions.  I don't think it's the same thing... and since this supports undefined that sounds best to me (unless undefined has some weird specific meaning)

Reply:
That makes sense. Changing it to undefined.